src/BlazorUI/Bit.BlazorUI.Extras/Components/RichTextEditor/BitRichTextEditor.scss

```
@@ -1,102 +1,394 @@
 .bit-rte-edt {
-    flex-grow: 1;
+    padding: spacing(1.7) spacing(2);
+    outline: none;
+
+    // Scroll wide content (notably wide tables) on both axes instead of clipping it, while
+    // keeping the existing vertical scrolling behavior of the editing surface.
+    overflow: auto;
+    line-height: 1.6;
+    color: $clr-fg-pri;
+    flex: 1;
+    // Allow this scroll pane to shrink below its content inside the .bit-rte flex column so its
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)